gui/freeadmin/static/lib/js/freeadmin.js

```diff
@@ -1416,6 +1416,19 @@ require([
 
     }
 
+    systemDatasetMigration = function() {
+        var dialog = new Dialog({
+            title: gettext("Warning!"),
+            id: "Warning_box_dialog",
+            content: domConstruct.create(
+                "p", {
+                    innerHTML: gettext(
+                        gettext("Warning: The action will result in migration of dataset.<br />Some services will be restarted.<br /><br />")
```

Comment:
I guess you did not understand me.

First, you are using gettext inside a gettedt, which is wrong. Second you are still using html tags within gettext...


Should be... gettext("Warning: The action will result in migration of dataset.") + "<br/>" + gettext(".....");